src/Control/Monad/Freer.hs

```diff
@@ -217,165 +217,21 @@ module Control.Monad.Freer
   , run
   , runM
 
-    -- ** Building Effect Handlers
```

Comment:
I chopped these out on the assumption that most people only want to use existing effects, not define new interpretations.

Reply:
"Most people" is poorly phrased. I meant "most modules" want only to use effects, many fewer want to interpret them.